Wiki/FAQ.md

```diff
@@ -0,0 +1,709 @@
+that project.
+
+There are also overrun CPU projects or projects that do not provide
+constant work unit (WU) flow.
+
+Try different projects and see whether your hardware can keep up a high
+magnitude. You can review your outcome magnitude in the Neural Network.
+Go to Advanced -\> Neural Network and search for your CPID. Then you can
+double click to drill into your participation or have a look here:
+<http://www.gridresearchcorp.com/gridcoin/?result&t=CPID_Leaderboard>
+
+Anyway, if you like a project you should support it, to keep the BOINC
+spirit up ;) I like the LHC and everything concerning CERN so I keep
+participating although it's not really worth it regarding GRC because
+they sometimes have an unstable WU feed.
+
+Additionally, you might want to re-visit the projects you contribute to,
+you don't want to contribute CPU power to a project that allows GPUs its
+like trying to mine Bitcoin with your PC, you are competing with the
+wrong equipment. You might start by visiting this list:
+<http://wiki.gridcoin.us/Current_BOINC_Whitelist> and make sure you
+aren't using your CPU for projects that have the ATI or Nvidia icon. You
+can also maximize your magnitude by finding projects that have smaller
+team RAC (just make sure they have work available) on the network report
+which you can find here:
+<http://www.gridresearchcorp.com/gridcoin/?result&t=Whitelisted_Project_Leaderboard>
+
+# Staking
+
+## How is the amount of stake-reward calculated?
+
+The stake reward is calculated by two combined
+algorithms.
+
+`Proof of Stake rewards will give you 10 GRC for each block you stack`  
+`Proof of Research rewards for BOINC contribution, though research activity. A PoR block is a combined payment 10 GRC and research reward.`
+
+## The wallet says "Not Staking because you don't have mature coins" , how long does it take for coins to mature?
+
+It takes about 16 hours for GRC to become mature from the time of
+arrival in your wallet. After a specific set of coins have Staked, they
+will need to mature again for 16 hours before they can Stake again.
+
+## Why can't I send coins in my wallet balance?
+
+You may have a balance reserved in stake. If this ever happens to you do
+this in the console:
+
+`reservebalance true AMOUNT`
+
+Where AMOUNT is the amount you want to send
+
+Wait a few blocks
+
+Then send
+it.
+
+## When I get an interest/mining payment, a portion of my GRC looks to be moving from my total balance into the "Stake" line of the wallet balance. Is that correct?
+
+Every 4 hours the wallet is on-line, it looks for old coins that can
+accrue interest and moves them from 'Balance' to 'Stake' then it works
+on staking. Once staked, it moves the staked balance back to
+balance.
+
+## I lose my stakeweight after receiving PoR? The "maturecoin-creationprocess" is starting again after each mined PoR?
+
+Yes that is normal, because your node builds up a set of stakeable coins

```

Comment:
Also not sure this is acurate

Reply:
This needs to be removed. It is not true.